import { type Dict } from './en';

const fmtErr = (msg: string) => `${msg} 请重试或检查控制台以获取更多信息。`;

export const zh: Dict = {
    common: {
        language: '语言',
        colorScheme: '主题',
        light: '明亮',
        dark: '黑暗',
        system: '系统',
        dashboard: '仪表板',
        users: '用户',
        admins: '管理员',
        login: '登录',
        logout: '登出',
        username: '用户名',
        password: '密码',
        confirmPassword: '确认密码',
        submit: '提交',
        save: '保存',
        confirm: '确认',
        cancel: '取消',
        email: '邮箱',
        phone: '电话',
        skills: '技能',
        hobbies: '爱好',
        id: 'ID',
        create: '创建',
        update: '更新',
        delete: '删除',
    },
    auth: {
        welcome: '欢迎来到 CDNet 👋🏻',
        errorAuthentication: fmtErr('在身份验证期间发生了错误。'),
        loggingOut: '登出中...',
        loggedOut: '您未登录。请登录以继续。',
        successLogin: '成功登录为 {{username}}',
        invalidCredentials: '登录失败，请检查。',
    },
    users: {
        errorGetUsers: fmtErr('无法检索用户。'),
        errorGetUser: '$t(users.errorGetUser)',
        createUser: '创建用户',
        createUserSuccess: '用户 {{username}} 已创建。',
        createUserError: fmtErr('在创建用户期间发生了错误。'),
        updateUser: '更新用户 {{username}}',
        updateUserSuccess: '用户 {{username}} 已更新。',
        updateUserError: fmtErr('在更新用户期间发生了错误。'),
        deleteTitle: '删除用户 {{username}}?',
        deleteMessage: '您确定要删除用户 {{username}} 吗？此操作不可逆。',
        deleteSuccess: '用户 {{username}} 已被删除。',
        deleteError: fmtErr('在删除用户期间发生了错误。'),
    },
    footer: {
        c1: '出自用',
        c2: '制作的',
    },
};
